### DISCUSSION TITLE
How to create a simple anchor link with Blade?

### DISCUSSION THREAD

**Discussion by nivv** [statamic/cms#7244] [Q&A]
I've been eyeing the link tag (https://statamic.dev/tags/link). In my blueprint I've added the `link` field and linked it another collection item. 

I'm trying to link to a section on the same page with an anchor link as in: `<a href="#services">Services</a>` where the anchor link is the slug of the linked entry.

This is what `$block->fileData()` prints:

```php
array:10 [▼ // resources/views/blocks/HeroBlock.blade.php
  "id" => "b00a499b-10b4-4f7f-888e-744bcec18213"
  "blueprint" => "block"
  "title" => "NIce title"
  "template" => "blocks/HeroBlock"
  "updated_by" => 1
  "updated_at" => 1671270495
  "image" => array:2 [▶]
  "block_link" => "entry::43507de6-92d7-4e40-84fa-f278f3d9d5ff" <---- Linked entry here
  "test_image" => array:2 [▶]
  "content" => "Some text"
]
```

When I do `{{ $block->link }}` it prints nothing. 

I'm probably missing something! Any ideas?

**Comment by nivv** [comment]
Closest I've come to a solution is this:

```blade.php
Statamic::tag('collection:blocks')->param('id:contains', $block->fileData()['block_link'])->fetch()
```
But that seems a bit convoluted, and if that is to work I have to remove the `entries::` portion of the string to find the entry.

`$block->fileData()['block_link']` outputs a string like below:

```
entry::43507de6-92d7-4e40-84fa-f278f3d9d5ff
```

**Comment by duncanmcclean** [comment]
I think you should be able to simply do `$block->block_link` which should resolve the link URL for you.

**Reply by nivv** [reply]
Thanks for your reply! I've tried that now and it does print `entry::43507de6-92d7-4e40-84fa-f278f3d9d5ff`, not an actual URL though. 

```
<a href="{{ $block->block_link }}">Test</a>
```
Output:
```
<a href="entry::43507de6-92d7-4e40-84fa-f278f3d9d5ff">Test</a>
```

**Reply by duncanmcclean** [reply]
What is block_link? A link field type?

What is $block - an entry or something else?
